demo-shell/src/app/components/card-view/card-view.component.html

```diff
@@ -38,6 +39,13 @@ <h3>Changes log:</h3>
         (change)="toggleNoneOption()"
         [checked]="showNoneOption">
         Show none option
+    </mat-slide-toggle><br>
+    <mat-slide-toggle
+        id="app-toggle-label-multivalued-chip"
+        [color]="'primary'"
+        (change)="toggleLabelForChips()"
+        [checked]="showLabelForChips">
+        Show label for multivalued (chips) property
```

Comment:
I missed that... `Show label for chips property`?

Reply:
Renamed